webpack.config.js

```diff
@@ -46,7 +88,8 @@ module.exports = {
         OPTIMIZELY_ACTIVE: process.env.OPTIMIZELY_ACTIVE,
         FULL_SUBDOMAIN_FOR_COOKIE: process.env.FULL_SUBDOMAIN_FOR_COOKIE,
         PAYPAL_EMAIL: process.env.PAYPAL_EMAIL,
-        PAYPAL_ENDPOINT: process.env.PAYPAL_ENDPOINT
+        PAYPAL_ENDPOINT: process.env.PAYPAL_ENDPOINT,
+        NODE_ENV: '"production"'
```

Comment:
Don't understand this one yet, but it was recommended it. It's perf boost didn't blow me away.